src/core/processing/qgsprocessingcontext.cpp

```diff
@@ -65,22 +65,22 @@ void QgsProcessingContext::setInvalidGeometryCheck( QgsFeatureRequest::InvalidGe
   mInvalidGeometryCallback = defaultInvalidGeometryCallbackForCheck( check );
 }
 
-std::function<void ( const QgsFeature & )> QgsProcessingContext::invalidGeometryCallback( QgsFeatureSource *source ) const
```

Comment:
These shouldn't have to change -- they can all still use const references

Reply:
Ah right, I'm never too at easy with pointers, ref and const. I never know if it's the reference value/address that must stays constent or the pointed object. I assume it would be the former and that the object may be changed but I didn't want to take the risk and wasn't sure what I needed to change to resolve the build error.